core/src/main/java/org/sonarsource/sonarlint/core/container/connected/update/ModuleConfigUpdateExecutor.java

```diff
@@ -120,6 +117,11 @@ private void updateStatus(Path temp) {
     ProtobufUtil.writeToFile(updateStatus, temp.resolve(StorageManager.UPDATE_STATUS_PB));
   }
 
+  private void fetchModuleHierarchy(String moduleKey, ModuleConfiguration.Builder builder) {
+    Map<String, String> moduleHierarchy = moduleHierarchyDownloader.fetchModuleHierarchy(moduleKey);
+    moduleHierarchy.forEach((key, path) -> builder.getMutableModulePathByKey().put(key, path));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1612)